####################
# Guidance message
####################

int32 guide_status
int32 moving_status
action[] actions
uint64 heading_node_id
float64 relative_angle
float64 distance_to_remain #remaining distance
string msg

